packages/toolkit/src/query/core/buildThunks.ts

```
@@ -430,6 +433,17 @@ In the case of an unhandled error, no tags will be "provided" or "invalidated".`
   const hasMaxAge = (
     options: any
   ): options is { ifOlderThan: false | number } => 'ifOlderThan' in options
+  const hasSubscribe = (options: any): options is { subscribe: boolean } =>
+    'subscribe' in options
+  const handlePrefetchSubscription = (
+    result: QueryActionCreatorResult<any>,
+    subscribe: boolean
+  ): void | QueryActionCreatorResult<any> => {
+    if (subscribe) {
+      return result
+    }
+    result.unwrap().finally(result.unsubscribe)
```

Should this catch and swallow the error?